torch/_inductor/ir.py

```diff
@@ -5826,6 +5826,11 @@ def codegen(self, wrapper) -> None:  # type: ignore[no-untyped-def]
                         equal_to_1.append(idx - index_shift)
 
                 triton_meta["configs"][0].equal_to_1 = equal_to_1
+        else:
+            constexpr_indices = []
+            for idx, kwarg in enumerate(self.ordered_kwargs_for_cpp_kernel):
+                if triton_meta['signature'][kwarg] == "constexpr":
```

Comment:
`kernel.constexprs` does not include all `constexprs` parameters that are in `triton_meta['signature']`